manager/src/test/java/com/redhat/service/bridge/manager/BridgesServiceTest.java

```diff
@@ -193,4 +197,35 @@ public void getBridge() {
     public void getBridge_bridgeDoesNotExist() {
         assertThatExceptionOfType(ItemNotFoundException.class).isThrownBy(() -> bridgesService.getBridge("foo"));
     }
+
+    @Test
+    public void testDeleteBridge() {
+        BridgeRequest request = new BridgeRequest(TestConstants.DEFAULT_BRIDGE_NAME);
+        bridgesService.createBridge(TestConstants.DEFAULT_CUSTOMER_ID, request);
+
+        //Wait for Workers to complete
+        Bridge bridge = TestUtils.waitForBridgeToBeReady(bridgesService);
+
+        bridgesService.deleteBridge(bridge.getId(), bridge.getCustomerId());
+
+        Bridge retrievedBridge = bridgesService.getBridge(bridge.getId(), TestConstants.DEFAULT_CUSTOMER_ID);
+        assertThat(retrievedBridge.getStatus()).isEqualTo(ManagedResourceStatus.DELETED);
+    }
```

Comment:
I think I'm missing a bit why this test works. `bridgesService.deleteBridge` should mark the Bridge with `DEPROVISION`

Reply:
I agree with @r00ta unless the Shard is running the `Bridge` should only be in `DEPROVISION` status. L#206 will wait for the `WorkManager` to execute the `Work` to provision the `Bridge` but its `status` will remain `ACCEPTED` and its `dependencyStatus` will be `READY` (as this test is annotated with `@TestProfile(WorkerSchedulerProfile.class)`. However nothing should move the `Bridge` to `DELETED` in the `manager` module.

Reply:
testcase is updated. Could you please review it again.